x-pack/plugins/observability_solution/inventory/common/entities.ts

```diff
@@ -49,6 +60,8 @@ export const defaultEntityDefinitions = [
 
 export const defaultEntityTypes: EntityType[] = ['service', 'host', 'container'];
 
+export const defaultEntitySortField = ENTITY_LAST_SEEN;
```

Comment:
It's a bit confusing that we have two default sorting fields:

`DEFAULT_ENTITIES_SORT_FIELD` and `defaultEntitySortField`

Reply:
100%, I missed the defaultEntitySortField entirely. I will update and use this one

Reply:
https://github.com/elastic/kibana/blob/b594c5a304e6b14098ecc4c9a2335895e8b86c89/x-pack/plugins/observability_solution/inventory/common/utils/parse_identity_field_values_to_kql.ts#L17